eu.lucaweiss.lpa-gtk.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: eu.lucaweiss.lpa-gtk
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: lpa-gtk
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pcsc
+  - --share=ipc
+  - --device=dri
+cleanup: []
+modules:
+  - lib/blueprint-compiler.yaml
+  - name: pcsclite # Dependency of lpac
+    config-opts:
+      - --disable-libsystemd
+      - --disable-polkit
+      - --disable-documentation
+    sources:
+      - type: archive
+        url: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/2.3.1.tar.gz
+        sha256: 7fcb59f66a323f63cf1ab492579a57d899806835c52ba377af9ac57df68bf39b
+        x-checker-data:
+          type: anitya
+          project-id: 2611
+          stable-only: true
+          url-template: https://github.com/LudovicRousseau/PCSC/archive/refs/tags/$version.tar.gz
+  - name: lpac # Dependency of lpa-gtk
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/estkme-group/lpac/archive/refs/tags/v2.2.1.tar.gz
+        sha256: 3d87080a625b10430eebb82f89e2d24e16a84a8435a9c40b3718fd88c82028ba
+        x-checker-data:
+          type: anitya
+          project-id: 371628
+          stable-only: true
+          url-template: https://github.com/estkme-group/lpac/archive/refs/tags/v$version.tar.gz
+  - name: lpac-logo # As long as lpa-gtk doesn't have it's own
```

Comment:
this should be done upstream then.

Reply:
Yeah, definitely don't take the eSTK.me logo and put it on my app, I don't know about copyright but it might also be trademarked

Reply:
Done